Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM ruby:2.5.3
+FROM ruby:2.5.3-alpine
 
 ENV APP_HOME /app
 RUN mkdir $APP_HOME
 WORKDIR $APP_HOME
 
+RUN apk add --update build-base postgresql-dev tzdata
```

Comment:
I think it's good practice to remove the packages once they're not needed, for example:

https://github.com/coderjoe/dockerfile-alpine-rails/blob/master/alpine-rails/Dockerfile

Reply:
hmmm... not too sure what you meant, 


pair these up with the gem `build-base` `postgresql-dev` `tzdata`?

or this `rm -rf /var/cache/apk/*`

?

Reply:
Sorry, that linked-to Dockerfile isn't the best example.

Yes, it's partly about the cache, but also the unused packages that are installed. For example, when building this Docker container we see `g++` et al being installed:

```
(12/26) Installing gcc (6.4.0-r9)
(15/26) Installing g++ (6.4.0-r9)
(16/26) Installing make (4.2.1-r2)
```

these aren't needed at runtime so they should also be removed. Here's a better reference that has a sample `Dockerfile`:

https://www.chrisblunt.com/rails-on-docker-getting-started-docker-ruby-rails/